config/settings_schema.json

```diff
@@ -989,8 +949,8 @@
         "type": "range",
         "id": "media_shadow_blur_radius",
         "min": 0,
-        "max": 20,
-        "step": 2,
+        "max": 100,
+        "step": 5,
```

Comment:
I wonder if 100 is too large of a blur radius.  Maybe not, but it is massive.  I'm okay with it, but I am curious if folks will actually use it.